content/guide/15-contributing-guide.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Contributing
+---
+
+### First time contributors
+
+#### Finding square one
+
+Wtih a large codebase, especially one for a compiler, it's challenging to now where to start contributing. The first step is to check out the [issue page](https://github.com/sveltejs/svelte/issues) for Svelte. It's recommended to skim through some of the `Bug` labeled issues to see if any grab you attention. As of now, we don't have a label for first time contributors but a `First Time Contributor` label is being discussed to be included in the triage of issues.
```

Comment:
now -> know
issue page -> issues page
you attention -> your attention

Reply:
Done, thanks!